.github/workflows/prerelease_staged_publish.yml

```diff
@@ -30,6 +30,7 @@ jobs:
   publishing-to-s3-linux:
     name: Publish linux artifacts into s3 staging bucket
     runs-on: ubuntu-20.04
+    if: ${{ (github.event.inputs.assetsType == 'all' || github.event.inputs.assetsType == matrix.assetsType) && (github.event.inputs.targzOS == 'all' || github.event.inputs.targzOS == matrix.targzOS) }}
```

Comment:
does this work? We will need to test it with staging/testing bucket after merge to master.

Reply:
cannot test until merged